### ISSUE TITLE
Sync settings to Cloud as well

### ISSUE THREAD

**Issue by claell** [Laverna/laverna#308]
I would like to have the ability to also sync the settings to the cloud. Might not be always helpful, though.

**Comment by varod** [comment]
it could be risky to sync encryption settings ;)

**Comment by claell** [comment]
This is true. But maybe one could sync all settings apart from encryption. (Probably it would be good to tell the user that encryption is not synced, so he does not delete it).
